client/src/partials/inventory/configuration/groups/groups.html

```diff
@@ -0,0 +1,54 @@
+<div class="panel panel-default">
+
+  <div class="flex-heading">
+    <span class="flex-heading-label">
+      <i class="fa fa-object-group"></i>
+      {{ 'INVENTORY.LIST_GROUP' | translate }} ({{ GroupsCtrl.groupList.length }})
+    </span>
+
+    <button
+      ng-click="GroupsCtrl.addInventoryGroup()"
+      class="pull-right btn btn-primary btn-sm"
+      type="button" data-create-group>
+      <i class="fa fa-plus"></i> {{ 'FORM.LABELS.ADD' | translate }}
+    </button>
+  </div>
+
+  <section style="height: 300px; overflow: auto;">
+    <table class="table table-condensed">
+      <thead>
+        <tr>
+          <th>{{ 'FORM.LABELS.CODE' | translate }}</th>
+          <th>{{ 'FORM.LABELS.NAME' | translate }}</th>
+          <th>{{ 'INVENTORY.CONTAINS' | translate }}</th>
+          <th class="text-center" colspan="3">{{ 'FORM.LABELS.ACCOUNT' | translate }}</th>
+          <th>&nbsp;</th>
+        </tr>
+        <tr>
+          <th></th><th></th><th></th>
```

Comment:
This is really difficult to understand if you have forgotten something by accident, or if you really mean to have three empty columns.  If you mean to have three empty columns, it is better to use:
```html
<th colspan="3"></th>
```
which looks identical in the rendered HTML.